operator/gitops/argocd/pipeline-service/tekton-results/kustomization.yaml

```diff
@@ -13,10 +13,10 @@ resources:
 images:
   - name: ko://github.com/tektoncd/results/cmd/api
     newName: quay.io/redhat-appstudio/tekton-results-api
-    newTag: bae7851ff584423503af324200f52cd28ca99116
+    newTag: dc54f48f6a1adfde07b923129db33aaa77a61213
```

Comment:
this tag exists over at https://quay.io/repository/redhat-appstudio/tekton-results-api?tab=tags